lib/widget/dropdown.dart

```diff
@@ -209,15 +210,12 @@ class SelectOneState extends FormFieldWidgetState<SelectOne> {
     if (items != null) {
       results = [];
       for (SelectOneItem item in items) {
-        item.iconname != null
+        item.icon != null
             ? results.add(
```

Comment:
So if first item has an icon and second item doesn't, the alignment will be off right? What I meant is:

First see if there is at least one item with icon specified, if so always add the gap for all items.

Reply:
okay